drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -222,7 +222,7 @@ static int _set_state(at86rf2xx_t *dev, netopt_state_t state)
             at86rf2xx_set_state(dev, AT86RF2XX_STATE_TRX_OFF);
             break;
         case NETOPT_STATE_SLEEP:
-            at86rf2xx_set_state(dev, AT86RF2XX_STATE_SLEEP);
```

Comment:
Do we even need `AT86RF2XX_STATE_SLEEP` anymore?

Reply:
maybe not